mathesar_ui/src/api/upgrade.ts

```diff
@@ -0,0 +1,11 @@
+export const upgradeApi = {
+  /** @throws Error for some very limited failure scenarios. */
+  async upgrade(): Promise<void> {
+    const response = await fetch('/api/ui/v0/upgrade/', {
+      method: 'POST',
+    });
+    if (!response.ok) {
+      throw new Error('Unable to upgrade.');
+    }
```

Comment:
This request will not succeed because there is no CSRF token here.

1. I have the same comment as earlier. I'd like to use the existing methods, or add another common method to `requestUtils` to make requests.
1. It's easier in the long run to handles all requests from a single file. eg., It'll help if we want to adding common request headers, log requests, be able abort them with CancellablePromise, handle errors in a common manner etc.,

Reply:
Fixed in b13a9e316cb5a3185de43e25a821b6538f22ded0.